### PR TITLE
Added 3 participants

### PR DESCRIPTION
**Pull request by ouphi**
# Added 3 participants to the Dataspace
- Added a `participants.json` containing the participants name

## Follow up story
Right now, if a user updates the `participants.json`, it will break the Verify step because Verify uses outputs from the deploy job with hardcoded names. Participants names are mentioned in different places and it would be better if they are mentioned in only one place. I created a [Spike](https://github.com/agera-edc/MinimumViableDataspace/issues/125) to investigate how to fix the problem.

### CODE REVIEW

File: .github/workflows/deploy.yaml [hunk 1]
```diff
@@ -175,3 +191,15 @@ jobs:
           CONSUMER_KEY_VAULT: ${{ needs.Deploy.outputs.company1_key_vault }}
           CONSUMER_CATALOG_URL: http://${{ needs.Deploy.outputs.company1_edc_host }}:8181/api/federatedcatalog
           API_KEY: ${{ needs.Deploy.outputs.company1_api_key }}
+
+      - name: 'System tests company 2 - company 3'
```
Comment (algattik) [inline .github/workflows/deploy.yaml]: why not a matrix with [company2, company3]?

**Review by algattik — CHANGES_REQUESTED** [review]
Comments inline

File: .github/workflows/deploy.yaml (hunk 1 above)
Comment (Izzzu) [inline .github/workflows/deploy.yaml]: Why we need 2 sets of system tests? In this case all the companies are the same so wouldn't it be enough that we have 1 System tests run?

Comment (Izzzu) [reply .github/workflows/deploy.yaml]: If it's to have a smoke test to check the set up maybe we can use 1 specific test and not the whole suite?

Comment (ouphi) [reply .github/workflows/deploy.yaml]: Can you explain what would be the value?

Comment (ouphi) [reply .github/workflows/deploy.yaml]: Do you mean only testing company1 and company 2 but not company3 ?

Comment (algattik) [reply .github/workflows/deploy.yaml]: So to run system tests of company1 vs company2, and company1 vs company3, you could create a matrix with [company2, company3].

Comment (ouphi) [reply .github/workflows/deploy.yaml]: I removed the second test.

Comment (ouphi) [reply .github/workflows/deploy.yaml]: I removed the second test.

**Review by algattik — APPROVED** [review]

**Comment by cpeeyush** [comment]
@ouphi PR description mentions `Added a system test involving company2 and company3`. But I do not see it in PR diff. is it missing ?

**Comment by ouphi** [comment]
> @ouphi PR description mentions `Added a system test involving company2 and company3`. But I do not see it in PR diff. is it missing ?

@cpeeyush, I got comments about removing this test because company3 has the same setup as company1 and company2 so we agreed to remove it. Thanks for letting me know I updated the description now.

**Review by Izzzu — APPROVED** [review]

**Review by cpeeyush — APPROVED** [review]

**Pull request merged**